cmd/ore/equinixmetal/equinixmetal.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2017 CoreOS, Inc.
```

Comment:
Let's update the license to update the license here.

Reply:
Thanks, since it's a brand new file, header has been updated to:
```
// Copyright The Mantle Authors
// SPDX-License-Identifier: Apache-2.0
```

Reply:
I wouldn't say it's a new file - just a rename of an old one…

Reply:
The line `Copyright 2017 CoreOS, Inc.` must stay